phonebook.json

```diff
@@ -0,0 +1,24 @@
+[{
```

Comment:
I suggest to add more entries (>20) to the example JSON file to demonstrate pagination capabilities of your program.